packages/gatsby-theme-try-ghost/src/templates/index.js

```diff
@@ -17,7 +17,8 @@ import { GlobalStateContext } from "../context/GlobalState"
 */
 const Index = ({ data, location, pageContext }) => {
     const posts = data.allGhostPost.edges
-    const { action } = location.search && location.search.length > 0 && queryString.parse(location.search) || { action: `ssr` }
+    const parsedQuery = location.search && location.search.length > 0 && queryString.parse(location.search)
+    const { action } = parsedQuery.action ? parsedQuery : parsedQuery.ssr !== undefined ? {action:`ssr`} : false
```

Comment:
Hi, currently, before this pr, the banner is shown but visually hidden by the dark header background-color.
I am not sure what the ssr logic is for. If it is redundant I can remove it, please let me know. Thank you.

Reply:
The location object is empty during SSR, which is different from an empty location during client rendering. The flag is for distinguishing the two different empty states, so consumers can implement different actions based on those states.

Reply:
> so consumers can implement different actions based on those states.

not sure how this is currently working, and if I missed something

we are checking now for window, maybe you can add additional logic here?

https://github.com/styxlab/gatsby-theme-try-ghost/blob/b196e0577cf976dcbf7530fa31cb5d8b8c592140/packages/gatsby-theme-ghost-members/src/gatsby-theme-try-ghost/components/common/SubscribeSuccess.js#L9